README.md

```diff
@@ -175,6 +176,33 @@ npm install
 npm run test
 ```
 
+<h2 id="docker-local-development">
+  Local Development using Docker (Dev Containers)
+</h2>
+
+1. Install docker https://www.docker.com/.
+2. Install [Microsoft VS Code](https://code.visualstudio.com/Download)
+3. Clone this repo.
+4. Open this repo in VS Code.
+5. Install the Dev Container extension.
+    - Navigate to View -> Extension  -> Search for and install _Dev Containers_ (or "ms-vscode-remote.remote-containers")
+    - More information on [Dev Containers](https://code.visualstudio.com/docs/devcontainers/containers)
+6. Build the dev container.
+    - Make sure you are connected to the Cerner VPN when building this container the first time.
+    - Navigate to View -> Command Palette -> Open Workspace In Container or Reopen In Container.
+7. You're now running in a dev container.  Use the terminal of the dev container to issue any npm commands.
+8. Before running any WDIO tests (`npm run test` or `npm run wdio`), make sure to perform the following steps:
+    - Open a new terminal (outside the dev container) and navigate to  ".devcontainer/" path in your repository.
+    - Execute the command `"docker compose -f docker-compose-wdio.yml up"`. Selenium hub should spin up. Leave this running in the background. If you see errors saying "container name already exists", run `"docker container prune"` command followed by selecting "y" to clear up any unused containers and try running the previous command again.
+    - Edit the root package.json file WDIO scripts to include `--disableSeleniumService=true` flag. This will disable the selenium service from spinning up again.
```

Comment:
Instead of this, maybe we can add a `docker:wdio` script that uses this flag?

Reply:
I will add in a new scripts `wdio:docker` and `test:docker` which would use the disableSeleniumService flag set to true.  I would like to still keep this explanation in the readme so they understand the process.